prospective.csv

```diff
@@ -26,4 +26,100 @@ translationstring,translationstring-feedstock,https://github.com/AnacondaRecipes
+hstspreload,,,,,"A script didn't find this package on any aggregate feedstock."
+ipaddress,,,,,"A script didn't find this package on any aggregate feedstock."
+jupyterhub-base,,,,,"A script didn't find this package on any aggregate feedstock."
+jupyterhub-singleuser,,,,,"A script didn't find this package on any aggregate feedstock."
+krb5-static,,,,,"A script didn't find this package on any aggregate feedstock."
+lapack,,,,,"A script didn't find this package on any aggregate feedstock."
+libblas,,,,,"A script didn't find this package on any aggregate feedstock."
+libcblas,,,,,"A script didn't find this package on any aggregate feedstock."
+libclang-cpp10,,,,,"A script didn't find this package on any aggregate feedstock."
+libclang-cpp12,,,,,"A script didn't find this package on any aggregate feedstock."
+libgfortran4,,,,,"A script didn't find this package on any aggregate feedstock."
+liblapack,,,,,"A script didn't find this package on any aggregate feedstock."
+liblapacke,,,,,"A script didn't find this package on any aggregate feedstock."
+libllvm10,,,,,"A script didn't find this package on any aggregate feedstock."
+libllvm11,,,,,"A script didn't find this package on any aggregate feedstock."
+libllvm12,,,,,"A script didn't find this package on any aggregate feedstock."
+libllvm9,,,,,"A script didn't find this package on any aggregate feedstock."
+libmlir12,,,,,"A script didn't find this package on any aggregate feedstock."
+libnetcdf,,,,,"A script didn't find this package on any aggregate feedstock."
+libssh2-static,,,,,"A script didn't find this package on any aggregate feedstock."
+libtmglib,,,,,"A script didn't find this package on any aggregate feedstock."
+llvm-lto-tapi,,,,,"A script didn't find this package on any aggregate feedstock."
+lua,,,,,"A script didn't find this package on any aggregate feedstock."
+lua-resty-http,,,,,"A script didn't find this package on any aggregate feedstock."
+luajit,,,,,"A script didn't find this package on any aggregate feedstock."
+mercurial,,,,,"A script didn't find this package on any aggregate feedstock."
+modin-omnisci,,,,,"A script didn't find this package on any aggregate feedstock."
+mtq,,,,,"A script didn't find this package on any aggregate feedstock."
+mxnet-gpu_mkl,,,,,"A script didn't find this package on any aggregate feedstock."
+mxnet-gpu_openblas,,,,,"A script didn't find this package on any aggregate feedstock."
+mxnet-mkl,,,,,"A script didn't find this package on any aggregate feedstock."
+mysql-python,,,,,"A script didn't find this package on any aggregate feedstock."
+numpy-devel,,,,,"A script didn't find this package on any aggregate feedstock."
+omniscidb,,,,,"A script didn't find this package on any aggregate feedstock."
+opencv-suite,,,,,"A script didn't find this package on any aggregate feedstock."
+pickle5,,,,,"A script didn't find this package on any aggregate feedstock."
+proj4,,,,,"A script didn't find this package on any aggregate feedstock."
```

Comment:
I think `proj4` was the previous name of the current `proj.4-feedstock`. But I'm not 100% sure